docs/pages/server-access/guides/ssh.mdx

```diff
@@ -0,0 +1,226 @@
+
+This Guide describes common scenarios one is likely to encounter when using Teleport as an SSH server, as an SSH client, or both.
+
+<Figure
+  align="center"
+  bordered
+  caption="Teleport SSH agent forwarding."
+>
+  ![Example cluster arrangement](../../../img/server-access/teleport-ssh.svg)
+</Figure>
+
+## Choose your SSH solution
+
+SSH tools are now commonly distributed as clients, servers, or some combination of both.  
+
+Teleport is unopinionated and agnostic about which client and/or server you choose. Admins can select specific Teleport features (like [Session Recording](../../features/enhanced-session-recording.mdx) while opting to continue using some or all of their existing SSH solutions.
+
+Teleport features can be used to "wrap" or enhance existing SSH tools while retaining the original, underlying, SSH access flows already in place.
+
+<Admonition title="Tip" type="tip">
+  For an added layer of secrecy, the default Teleport listening ports can be overridden by port numbers of your choice using Teleport YAML configuration files.
+</Admonition>
+
+### Clients
+
+Teleport is compatible with popular clients like [OpenSSH](./openssh-client.mdx) and also provides its own [built-in SSH client](../../cli-docs.mdx#tsh) (`tsh`).
+
+By default, SSH clients can connect to a Teleport cluster through the following port (exposed through the public proxy):
+
+| Port | Service | Description |
+| - | - | - |
+| 3023 | Proxy | SSH port clients connect to. A proxy will forward this connection to port `#3022` on the destination Node. |
+
+Teleport allows for complex, multi-client, arrangments as well as unified solutions (using Teleport alone in both a dedicated SSH server and as a client through `tsh`). For example, one cluster may be accessible through OpenSSH, another through Teleport `tsh`, and another through Putty.
+  
+### Servers
+
+Teleport is agnostic concerning your choice of SSH server backends and can be used to supplement existing features or serve as an intermediary between them and public clients.
+
+A common strategy involves using Teleport Nodes in Proxy mode to get all the benefits of Audit Logging, Session Recording, and/or Two-Factor Authentication.
+
+Most backends expose port `22` for inbound public clients and `tsh` will connect through the specified `--port` value on command:
+
+```bash
+# tsh ssh into a public proxy Node using a specific port
+tsh ssh --port=22 user@tele.example.com
+```
+
+## Connect to a Teleport cluster using tsh ssh
+
+Teleport SSH is robust enough that you can do all the things you'd do with OpenSSH, without it.
+
+To `tsh` into a cluster at `tele.example.com` with user `tele-admin`:
+
+1. On your local machine, log in through `tsh`: 
+
+   ```bash
+   # Log in through tsh
+   tsh login --proxy=tele.example.com:443 --user=tele-admin
+   ```
+   
+   You'll be prompted to supply the password and One Time Passcode you set up previously for `tele-admin`.
+
+2. `tele-admin` will now see:
+
+   ```bash
```

Comment:
This is an example, not a necessary role assignment.

In the future, I think we should have a separate "example-only" code-block: https://github.com/gravitational/next/issues/275 clearly indicating that it's an example and can't be copy-pasted.